_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -168,6 +183,7 @@ cats
 > {: .solution}
 {: .challenge}
 
+#Realistic example
```

Comment:
I propose leaving this as a first-level header because it summarizes the entire lesson previously. No changes made.